Dataset.py

```diff
@@ -92,7 +92,7 @@ def __init__(self, name=None,
     :param str name: e.g. "train" or "eval"
     :param int window: features will be of dimension window * feature_dim, as we add a context-window around.
       not all datasets support this option.
-    :param None|int|dict|NumbersDict context_window: will add this context for each chunk
+    :param None|int|dict|NumbersDict|tuple|list context_window: will add this context for each chunk
```

Comment:
I would just write explicitly `(int,int)|(dict,dict)|(NumbersDict,NumbersDict)`. No need to specifically also mention list here, even if we handle it, and also not all the possible combinations (like `(int,dict)`, even though you should also handle that). But more importantly, it's better to specify that you expect two elements, not just a list/tuple of any length.

Reply:
Actually, maybe just supporting `(dict,dict)` is fine for now, to not make the code too complicated, for cases which you don't use currently.